README.md

```diff
@@ -1,65 +1,59 @@
-Commerce Vipps
---------------
+# Commerce Vipps
 
-This module is published on [Drupal.org][Commerce Vipps] and [github.com][Commerce Vipps on GitHub].
+This module is published on [Drupal.org][Commerce Vipps] and [gitub.com][Commerce Vipps on GitHub].
```

Comment:
Typo?

Reply:
UhOh! Thank you!

Reply:
Fixed. The lint editor was nagging me to capitalize the H and I did and meant to undo, but lost it instead.